plugin/helpers_kv.go

```diff
@@ -220,3 +220,25 @@ func (p *HelpersImpl) KVListWithOptions(options ...KVListOption) ([]string, erro
 
 	return ret, nil
 }
+
+func (p *HelpersImpl) KVModify(key string, f func(value []byte) ([]byte, error)) error {
+	err := p.ensureServerVersion("5.2.0")
+	if err != nil {
+		return err
+	}
+
+	v, appErr := p.API.KVGet(key)
+	if appErr != nil {
+		return appErr
+	}
+
+	modifiedValue, modifiedError := f(v)
+	if modifiedError != nil {
+		return err
+	}
+	setErr := p.API.KVSet(key, modifiedValue)
+	if setErr != nil {
+		return setErr
+	}
+	return nil
```

Comment:
Nit: There is no need for a new error variable
```suggestion
	}
	
	err := p.API.KVSet(key, modifiedValue)
	if appErr != nil {
		return appErr
	}
	
	return nil
```

Reply:
fixed. Thanks.

Reply:
Return back custom err name because of go vet proposal.